huh/db.py

```diff
@@ -81,6 +81,53 @@ def all(cls, conn: sqlite3.Connection):
         res = cur.execute(f"SELECT rowid AS id, * FROM {cls.table_name}", ())
 
         return (cls(*row) for row in res)
+    
+    @classmethod
+    def all_ann(cls, conn):
+        cur = conn.cursor()
+        cur.row_factory = sqlite3.Row
+        res = cur.execute(
+            """
+            SELECT Announcement.rowid,title,name,timestamp,content FROM 
+            Announcement JOIN User ON Announcement.author_id=User.rowid
+            """
+            )
+        return res.fetchall()  
+    
+    @classmethod
```

Comment:
use staticmethod decorator -- classmethod is only for static methods that need access to cls (the  properties on the static class), and cls seems to be unused here